artiq/firmware/satman/satman.ld

```diff
@@ -68,11 +81,11 @@ SECTIONS
         _fstack = . - 16;
     } > main_ram
 
-    /* 64MB heap for alloc use */
+    /* remainder of 64MB for heap for alloc use */
     .heap (NOLOAD) : ALIGN(16)
     {
         _fheap = .;
-        . = . + 0x4000000;
+        . = 0x44000000; // not to overwrite RPC queue
```

Comment:
I did notice that ``rpc_queue`` as defined in ``board_artiq`` begins at 0x44000000. Although in case of satman we could still overwrite that as subkernels cannot use RPCs at the moment, that seems like a potential trap if it was left as is.

Reply:
We could follow the runtime's format, with a define of the ``runtime`` memory area as ``runtime (RWX) : ORIGIN = 0x40000000, LENGTH = 0x4000000 /* 64M */``... the layout seems actually to be exactly the same now anyway.